.github/build-ci/manifests/mom5/spack.yaml.j2

```
@@ -0,0 +1,6 @@
+spack:
+  specs:
+  - mom5 %intel@2021.10.0 arch=x86_64
```

```suggestion
  - mom5 %{{ intel_compiler }} arch=x86_64
```
Can the non-default manifests access the standard_definitions? If so it should use that I think.

How could a package define it's own data file if there were several manifests they wanted to share common settings across?

If we want to apply `arch` consistently across all tests it should be in the standard_definitions and added to the default I think.

---

Yes they can, and in the case of this, they do - it just doesn't use the values it's given because we don't have any of the template vars in the template (yet).

For your second point, it would require some more logic in the `setup` job. We'd have to find a way to associate each matrix entry with a different data file, in a similar way that we associate a manifest with a package. It would be a bit more involved. Currently we associate every matrix entry with the same data file. 

For the last point, fair enough. I think I'll remove the arch for now. It was just to differentiate the template from the "specific" mom5 one. The mom5 manifest is there pretty much as an example for future specific package manifests.

---

Done in https://github.com/ACCESS-NRI/spack-packages/pull/276/commits/2fa28e51fee796d2b5deb644dd9b4f4b21ad9eee

---

> The mom5 manifest is there pretty much as an example for future specific package manifests.

Let's use an example we definitely want to do then. ACCESS-ISSM uses `+wrappers` in the build but it is off by default

https://github.com/ACCESS-NRI/access-issm/blob/main/spack.yaml#L9

https://github.com/ACCESS-NRI/spack-packages/blob/main/packages/issm/package.py#L40-L44

and we should also test the `+ad` variant.

Can you make a manifest and test those variants in a matrix within the manifest?